src/clientsettingsdlg.cpp

```diff
@@ -397,6 +397,11 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                         "A second sound device may be required to hear the alerts." ) );
     chbAudioAlerts->setAccessibleName ( tr ( "Audio Alerts check box" ) );
 
+    // show tooltips
+    chbShowToolTips->setWhatsThis ( "<b>" + tr ( "Show ToolTips" ) + ":</b> " +
```

Comment:
I believe it should be "Tooltips" by the style guide. 
@gilgongo ?

Reply:
The Guide says "Nouns use lower case unless they have a formal definition in this style guide". So it looks like if we're to be consistent with "check box" (for example) we might need to have it as "Show tool tips". 

As an aside, we also use capitals to refer to things unique or in some way special to Jamulus, and not only terms in the Guide (hence "Audio Alerts" for example). Might add that to the Guide in fact. I like to think our slightly unusual use of capitals is inherited from the German origins of Jamulus :-)